modules/ui/fields/input.js

```diff
@@ -167,9 +168,29 @@ export function uiFieldText(field, context) {
                     if (value) window.open(value, '_blank');
                 })
                 .merge(outlinkButton);
+        } else if (field.key.includes('colour')) {
```

Comment:
this works for the most common fields (`colour=*`, `roof:colour=*`, `building:colour=*`), but I think we should introduce a new field type to https://github.com/openstreetmap/id-tagging-schema, since this logic erroneously matches tags like `seamark:topmark:colour_pattern=*`

Reply:
Let's continue this in https://github.com/openstreetmap/id-tagging-schema/issues/284